front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/SurveyQuestionResultWidget/Question/Source.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+// api
+import usePhases from 'api/phases/usePhases';
+import useProjectById from 'api/projects/useProjectById';
+
+// components
+import { Text } from '@citizenlab/cl2-component-library';
+import Link from 'utils/cl-router/Link';
+
+// i18n
+import messages from '../messages';
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+import useLocalize from 'hooks/useLocalize';
+
+// utils
+import { generateDateRange, getPhaseIndex } from './utils';
+
+interface Props {
+  projectId: string;
+  phaseId: string;
+}
+
+const Source = ({ projectId, phaseId }: Props) => {
```

Comment:
This is the 'Data from x during y' part
![Screenshot 2024-02-07 at 11 20 45](https://github.com/CitizenLabDotCo/citizenlab/assets/13822005/3b8af222-9de2-4241-a55b-5d091626a05b)